DotNetHowToAutoComplete/DotNetHowToAutoComplete/packages.config

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Search.Common" version="5.0.0" targetFramework="net461" />
```

Comment:
You'll need to reference version 6-preview in order for this to work. You can test it locally by running `Build-SeachPackages.ps1`, and then make sure to include the binaries/packages directory in your local azure-sdk-for-net repo in your NuGet package sources.